### PR TITLE
[Bugfix-22723] Change pasteKey example syntax

### PR DESCRIPTION
**Pull request by livecodealice**
Changed the single line if statement to multiple lines so as to reduce confusion.

### CODE REVIEW

File: docs/dictionary/message/pasteKey.lcdoc [hunk 1]
```diff
@@ -18,9 +18,10 @@ Platforms: desktop, server
 
 Example:
 on pasteKey -- replace returns with spaces before pasting
-  if the clipboardData["text"] is not empty then set \
-     the clipboardData["text"] to \
+  if the clipboardData["text"] is not empty then 
+     set the clipboardData["text"] to \
      replaceText(the clipboardData["text"],return,space)
```
Comment (livecodepanos) [inline docs/dictionary/message/pasteKey.lcdoc]: This line should be indented, so as it appears like this:
```
if the clipboardData["text"] is not empty then 
   set the clipboardData["text"] to \
         replaceText(the clipboardData["text"],return,space)
end if
```

(You can paste it in the LC script editor to see how it appears there)

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok 3a143b7

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok b2651b4

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok b2651b4ed6727b79ce23066297e540c5e9035865

**Comment by livecode-vulcan** [comment]
😎 test success b2651b4ed6727b79ce23066297e540c5e9035865

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/803)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/212)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/212)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/212)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/212)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2329)
* try-community-universal-ios-iphoneos13.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.4/builds/62)
* try-community-universal-ios-iphonesimulator13.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.4/builds/64)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2357)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2345)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2344)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2379)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2319)

**Pull request merged**